extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -605,3 +605,9 @@ export enum PublishTargetType {
 }
 
 export const CollapseProjectNodesKey = 'collapseProjectNodes';
+
+// httpClient
+export const downloadError = localize('downloadError', 'Download error');
```

Comment:
For calls to localize the vscode convention is that the string ID parameter uses single quotes and the string constant uses double quotes.  There should be a linting rule for this.  Such as, `localize('downloadError', "Download error");`.  Though if these are for the output log they probably don't need to be localized.

Reply:
VS Code got rid of that linting rule a while back. I haven't looked into exactly why but I never really understood why they had that in the first place since ' and " are the same in Javascript.

Reply:
@Charles-Gagnon, I'm not sure why they had this convention, but they seem to follow it consistently.  If it's not showing up as warnings in the build I don't think we have any tooling dependency on this AFAIK, so probably safe to ignore.  Yes, beyond the syntactical difference they are the same in JS.

Reply:
oh interesting, I do remember getting errors before whenever I used single quotes instead of double quotes for localized strings. Changed them to double quotes for consistency.